graph/util.go

```diff
@@ -0,0 +1,21 @@
+package graph
+
+import (
+	"runtime"
+)
+
+// isAlphanumeric checks whether a particular rune is alphanumeric
+func isAlphanumeric(c rune) bool {
+	return (c >= 'a' && c <= 'z') || (c >= 'A' && c <= 'Z') || (c >= '0' && c <= '9')
+}
+
+func getTestGlobalDefaults() string {
+
+	var fileLoc string
+	if runtime.GOOS == "windows" {
+		fileLoc = "./global-defaults-windows.yaml"
+	} else { // Looking at Linux
+		fileLoc = "./global-defaults-linux.yaml"
+	}
+	return fileLoc
+}
```

Comment:
Can you check `util/sock_linux.go` and `util/sock_windows.go` to use platform-aware const variable?

Reply:
Should I be checking these paths ?. Not fully certain what these do

Reply:
remove loading aliases from yaml files.